inc/saiswitch.h

```diff
@@ -338,6 +338,8 @@ typedef enum _sai_tunnel_type_t
 
     SAI_TUNNEL_TYPE_MPLS,
 
+    SAI_TUNNEL_TYPE_SRV6,
```

Comment:
Right. So if we are using SRV6 here, why are we not calling it SRV6 other places where we are using SEGMENTROUTE

Reply:
Good point. When SRv6 support was added to SAI a few years ago, authors decided to use "SEGMENTROUTE" for some reason. If backward compatibility is not a concern, I can replace all instances of SEGMENTROUTE with SRV6.

Reply:
This topic was discussed in SRv6 workgroup and everyone agreed that backward compatibility is not a concern. I'll rename segmentroute to srv6 everywhere.